regression_tests/i356-enum-without-int-helpers/tests/bindings/test.kts

```diff
@@ -0,0 +1,4 @@
+import uniffi.regression_test_i356.*;
+
+assert(which(true) == Which.YEAH)
+assert(which(false) == Which.NAH)
```

Comment:
So I guess this wouldn't work before, but now it does? That's what makes it a useful test?

Reply:
Prior to the fix, we wouldn't even get to running this, it would fail to compile. Which, it's probably worth me adding a comment to that effect in the test here for clarity.